charts/victoria-metrics-cluster/templates/_helpers.tpl

```diff
@@ -45,61 +45,61 @@ Create the name of the service account
 {{/*
 Create unified labels for victoria-metrics components
 */}}
-{{- define "victoria-metrics.common.matchLabels" -}}
+{{- define "victoria-metrics.common.matchLabels" | trunc 60 -}}
```

Comment:
I don't think this is the right fix, we should limit the length of each lable value.

Reply:
thanks, you mean truncating it e. g. [here](https://github.com/VictoriaMetrics/helm-charts/blob/504c6d03f17eb9356ba23b7a1ee9fc6f07782ea0/charts/victoria-metrics-cluster/templates/vminsert-deployment.yaml#L34)

Reply:
@brandshaide It means truncating each label value separately. For example: https://github.com/VictoriaMetrics/helm-charts/blob/624f078a10be80669b0e48238ecd9bd5d049bd5a/charts/victoria-metrics-cluster/templates/_helpers.tpl#L49-L50

This could be updated to become:
```
app.kubernetes.io/name: {{ include "victoria-metrics.name" . | trunc 60 }}
app.kubernetes.io/instance: {{ .Release.Name | trunc 60 }}
```


Also, note that using filters in `define` is not allowed by Helm syntax. You can validate your changes locally by using `make lint`(make file [here](https://github.com/VictoriaMetrics/helm-charts/blob/624f078a10be80669b0e48238ecd9bd5d049bd5a/Makefile#L62-L71)).
By default, it uses docker to run linting, you can switch to local helm adding `HELM=helm-local` to the end of command: `make lint HELM=helm-local`

Reply:
@zekker6 thanks for your help, I've applied the changes (also did run linting without errors)